src/oic/oauth2/__init__.py

```diff
@@ -875,14 +880,16 @@ def do_access_token_request(
         state: str = "",
         body_type: ENCODINGS = "json",
         method="POST",
+        request=None,
```

Comment:
The same change should be done in `src/oic/oic/__init__.py` implementation of `Client`.

Also a test of this behaviour would be a nice addition.

Reply:
Done. Awaiting approval 🙂.